btf/ext_info.go

```diff
@@ -374,6 +384,21 @@ func newFuncInfos(bfis []bpfFuncInfo, spec *Spec) ([]FuncInfo, error) {
 	return fis, nil
 }
 
+// NewFuncInfos parses btf func info in wire format.
+func NewFuncInfos(reader io.Reader, bo binary.ByteOrder, recordNum uint32, spec *Spec) ([]FuncInfo, error) {
```

Comment:
Nit: other constructors from a reader in this package are `Load*`. Should these be `LoadFuncInfos`?